salt/search/config/etc-syslog-ng-conf.d-search.conf

```diff
@@ -0,0 +1,43 @@
+@version: 3.5
+
+source s_search_application_debug {
+    file("/srv/search/var/logs/debug.log"
```

Comment:
I think so, I'm using the line formatter here:
https://github.com/elifesciences/search/pull/55/commits/890f7bc614aaf0009ca5c362ef9a77b1b54fc363#diff-86edef8de801fbe6d5b41bc89aa32d13R187